pom.xml

```diff
@@ -309,6 +309,14 @@
                 <artifactId>maven-pmd-plugin</artifactId>
                 <version>3.8</version>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <configuration>
+                    <configLocation>config/sun_checks.xml</configLocation>
```

Comment:
First of all, it's a wrong config file. Secondly, we already do `checkstyle` on the build step.